src/styles/components/_product-card.scss

```diff
@@ -0,0 +1,31 @@
+.product-card {
+  display: inline-block;
+  width: 50%;
+  height: rem(200px);
+  padding: rem(20px 8px);
+  box-sizing: border-box;
```

Comment:
According to [this ref](https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/), please move this below your `display: inline-block`